data/FedexDemoData.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<entity-facade-xml type="demo">
+    <mantle.party.Party partyId="Fedex Party" partyTypeEnumId="PtyPerson"/>
```

Comment:
These are not the demo data

Reply:
While shifting Party and Enumeration Data to seed-initial, ShippingGatewayConfig Data is showing referential Integrity Error due to shippingGatewayTypeEnumId.